index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Template 3</title>
+</head>
+<body>
+    <header class="topbar">
+        <h3>Welcome</h3>
+        <nav class="menu"><img src="./images/icons/Union.png" alt="menu"></nav>
+    </header>
+    <ul class="main">
+        <li class="st">
+            <h1 class="t1">Hey there. I’m Sameh</h1>
+            <h1 class="t2">I’m a software developer</h1></li>
+        <li><p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p></li>
+        <li class="socialIcons">
+            <img src="./images/icons/twitter.png" alt="twitter">
+            <img src="./images/icons/Linkedin.png" alt="linkedin">
+            <img src="./images/icons/M.png" alt="m">
+            <img src="./images/icons/git.png" alt="git">
+            <img src="./images/icons/2.png" alt="2">
+        </li>
```

Comment:
- [ ] Please wrap the social media links with a ul that has multiple links for each social media link. You can use something like the following:
```
<ul>
   <li>
      <a>
         <img> first image
      </a>
   </li>
   <li>
      <a>
         <img> second image
      </a>
   </li>
   ...
</ul>
```